### PR TITLE
Setup updates

### PR DESCRIPTION
**Pull request by jonah-williams**
Setting up a new machine identified several places where the setup script could be improved. See individual commits.

### CODE REVIEW

**Comment by bobzoller** [comment]
thanks @jonah-williams

File: setup.sh [hunk 1]
```diff
@@ -89,12 +89,17 @@ brew cask install google-chrome firefox google-hangouts screenhero
...
 # mongo
+brew install qemu
+if [ -f /Applications/Docker.app/Contents/Resources/moby/data.qcow2 ]; then
+  rm /Applications/Docker.app/Contents/Resources/moby/data.qcow2
+fi
+qemu-img create -f qcow2 /Applications/Docker.app/Contents/Resources/moby/data.qcow2 120G
 echo "Setting up docker containers for mongo to match production mongo clusters"
-docker run --name mongo-shopping -d --restart always -v mongo-shopping-data:/data/db -p 27017:27017 mongo:2.6.9
-docker run --name mongo-ops -d --restart always -v mongo-ops-data:/data/db -p 27018:27017 mongo:3.2.8
+[[ $(docker ps -f 'name=mongo-shopping' --format '{{.Names}}') == 'mongo-shopping' ]] || docker run --name mongo-shopping -d --restart always -v mongo-shopping-data:/data/db -p 27017:27017 mongo:2.6.9 --smallfiles
```
Comment (sylspren) [inline setup.sh]: oh oops! I forgot to update this after the mongo upgrade. Do you mind also changing the shopping to `mongo:3.2.8`?

Comment (bobzoller) [inline setup.sh]: I was going to get this and enabling wiredtiger in a follow-on :)

Comment (bobzoller) [inline setup.sh]: 3.2.11 also

Comment (sylspren) [inline setup.sh]: tyty!

**Pull request merged**